packages/ui/src/working-groups/queries/workingGroups.graphql

```diff
@@ -25,6 +26,9 @@ fragment WorkerFields on Worker {
   hiredAtBlock {
     ...BlockFields
   }
+  application {
+    ...WorkingGroupApplicationFields
```

Comment:
~Maybe fetching application id & opening id would be enough?~

```
application {
  id
  opening {
    id
  }
}
```

Reply:
However, it would require a bigger change in the modal system so for now it is OK.